src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -167,14 +159,14 @@ default String checkTitleMatches(String expectedTitlePattern, double waitFor, do
      * @param timeTook     - the amount of time it took for wait for something (assuming we had to wait)
      * @return Boolean: whether or not the text is actually present
      */
-    default boolean checkTextPresent(String expectedText, double waitFor, double timeTook) {
+    boolean checkTextPresent(String expectedText, double waitFor, double timeTook) {
         // check for the object to be present
-        boolean isPresent = getApp().is().textPresent(expectedText);
+        boolean isPresent = this.app.is().textPresent(expectedText);
         if (!isPresent) {
-            getReporter().fail("to find text <b>" + expectedText + "</b> present on the page", waitFor, TEXT_B + expectedText + "</b> is not present on the page", timeTook);
+            this.reporter.fail("to find text <b>" + expectedText + "</b> present on the page", waitFor, TEXT_B + expectedText + "</b> is not present on the page", timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "</b> present on the page" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)